src/components/CaseStudies/StaffSpotlight.tsx

```diff
@@ -1,57 +1,56 @@
-interface Author {
+interface Staff {
     image?: string
     name: string
     title: string
 }
 
-export const AuthorBio: React.FunctionComponent<{
+export const StaffSpotlight: React.FunctionComponent<{
     customer: string
     about: string
-    authors: Author[]
-}> = ({ customer, about, authors }) => (
+    staff: Staff[]
+}> = ({ customer, about, staff }) => (
     <div>
-        {authors.length > 1 ? (
-            // Multi author design
+        {staff.length > 1 ? (
+            // Multi staff design
             <div>
                 <div className="tw-mx-auto tw-max-w-screen-sm">
                     <h5 className="mb-4">About {customer}</h5>
                     <p>{about}</p>
                 </div>
 
                 <div className="tw-grid tw-gap-6 tw-grid-cols-1 md:tw-grid-cols-3 tw-mt-16 tw-max-w-screen-lg tw-mx-auto">
-                    {authors.map(a => (
+                    {staff.map(a => (
                         <div key={a.name} className="md:tw-text-center">
                             <img
-                                className="tw-rounded-full max-w-150 tw-inline"
+                                className="tw-rounded-full max-w-150 tw-inline tw-mb-sm"
                                 width={150}
                                 height={150}
                                 src={a.image}
                                 alt={a.name}
                             />
-                            <h3 className="mt-4">{a.name}</h3>
-                            <p className="text-muted">{a.title}</p>
+                            <h3>{a.name}</h3>
+                            <p className="tw-text-gray-400">{a.title}</p>
                         </div>
                     ))}
                 </div>
             </div>
         ) : (
-            // Single author design
+            // Single staff design
             <div className="tw-flex tw-flex-col md:tw-flex-row">
-                {authors[0].image && (
-                    <div className="col-md-3">
+                {staff[0].image && (
+                    <div className="tw-mb-sm md:tw-mb-0 tw-px-0">
```

Comment:
Since you removed the `col-md-3`, the px-0 padding isn't needed since there's nothing to override 😅

Reply:
Fine tooth comb! Thx for [f203558](https://github.com/sourcegraph/about/pull/5706/commits/f2035586857373b7f9d70e927a94b207877ae58a)!